package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Looks like an accidentally committed file - can we delete it?

Reply:
Sure, will do